cnd/src/swap_protocols/rfc003/bitcoin/htlc_events.rs

```diff
@@ -68,7 +68,7 @@ where
             .expect("Deployment transaction must contain outpoint described in pattern");
 
         Ok(Deployed {
-            location: OutPoint {
+            location: ::bitcoin::OutPoint {
```

Comment:
```suggestion
            location: htlc_location::Bitcoin {
```
What do you think?

Reply:
Yes!

At some point we may even create our own wrapper types instead of just aliasing :)

Reply:
Nice, my brain only got to seeing that `Self::HtlcLocation` would not work.  Thanks